synapse/module_api/__init__.py

```diff
@@ -1520,6 +1521,26 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def get_room_alias(self, room_id: RoomID) -> Optional[RoomAlias]:
+        """
+        Get the room alias associated with a room ID.
+
+        Added in Synapse v1.82.0.
+
+        Args:
+            room_id: The Room ID to find the alias of.
+
+        Returns:
+            The optional room alias
```

Comment:
```suggestion
        None if the room ID does not exist, or if the room exists but has no canonical alias.
        Otherwise, the parsed room alias.
```

(I _think_ the return value is None if the room ID does not exist?)